frontend/buffers/editorAutocompletion.ts

```diff
@@ -167,12 +167,13 @@ function getSnippet(proto) {
 
 export const getFunctionsInfo = function (functionName, strings, ignoreDoc = false) {
     let blockDesc = '', funcProto = '', blockHelp = '';
-    const docGenerator = createDocGenerator(null, strings);
```

Comment:
Delete createDocGenerator from `frontend/common/docGenerator.ts` (which was code specific to blocklyProcessing and not code to be integrated into Codecast).

Maybe leave a comment here about integrating context.docGenerator later when we'll work on blocklyProcessing?